### ISSUE TITLE
TypeScript error for Reducer generic type

### ISSUE THREAD

**Issue by marclemagne** [supasate/connected-react-router#325]
Having trouble compiling TypeScript in my project because of this error:
```
ERROR in /Users/.../.../.../node_modules/connected-react-router/index.d.ts(96,7):
TS2314: Generic type 'Reducer' requires 1 type argument(s).
```
The type is described as:
```
export function connectRouter(history: History)
  : Reducer<RouterState, LocationChangeAction>
```
If I manually remove `LocationChangeAction` in the type file the application compiles and seemingly works fine. 

Is there something I'm doing wrong here?

NOTE: Previously using "react-router-redux" but here because it recommends this library.

Here are some relevant dependencies from `package.json`:
"connected-react-router": "^6.4.0",
"react-redux": "^7.1.0",
"react-router-dom": "5.0.1",
"react": "16.8.6"

I appreciate any help/advice. Thanks!

**Comment by rodrigopavezi** [comment]
@marclemagne how did you solve this issue?

**Comment by JakeBar** [comment]
@marclemagne I'd also be interested to know how you solved this

**Comment by qburst-sibil** [comment]
![image](https://user-images.githubusercontent.com/43597146/77312399-efa5e400-6d27-11ea-84e5-f80368990979.png)
@marclemagne the changes are not pushed in the code. it is still there

**Comment by gavinsharp** [comment]
This error occurred when using `combineReducers`; it was fixed in https://github.com/supasate/connected-react-router/pull/431 (released in [v6.9.0](https://github.com/supasate/connected-react-router/releases/tag/v6.9.0)).
